Project8/src/TransportSystem/Airplane.java

```diff
@@ -0,0 +1,14 @@
+package TransportSystem;
+
+public class Airplane implements Transport{
+    private int destination;
+
+    public Airplane() {
```

Comment:
do we really need default constructor here?

please remove it if not

Reply:
Fixed